lib/api/src/grpc/validate.rs

```diff
@@ -70,7 +70,20 @@ impl Validate for grpc::vectors_config::Config {
     fn validate(&self) -> Result<(), ValidationErrors> {
         use grpc::vectors_config::Config;
         match self {
-            Config::Params(params) => params.validate(),
+            Config::Params(params) => {
+                if params.multivector_config.is_some() {
+                    let mut err = ValidationErrors::new();
+                    err.add(
+                        "multivector_config",
+                        ValidationError::new(
+                            "Multivector configuration is not supported for anonymous vectors",
```

Comment:
Why do we disallow anonymous name? In sparse vectors we had REST API restrictions to do that. Here we extend dense vectors API and can do  anonymous names.

Moreover, it seems we allow to create anonymous names in rest:
https://github.com/qdrant/qdrant/blob/dev/lib/collection/src/operations/types.rs#L1590

Reply:
You are right, good catch!